server/backend/users/tests/test_views_reset_password.py

```
@@ -0,0 +1,58 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from django.core import mail
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+
+User = get_user_model()
+
+@pytest.mark.django_db
+def test_request_password_reset_view(client):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_request_password_reset_view`

Add return type annotation: `None`

(ANN201)